sdk/appconfiguration/azure-appconfiguration/setup.py

```diff
@@ -87,7 +87,7 @@
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
         "msrest>=0.6.10",
-        "azure-core<2.0.0,>=1.0.0b5",
+        "azure-core<2.0.0,>=1.0.0",
```

Comment:
Will setting the core packages to 1.0.0 and updated the dependencies in packages that depend on core cause problems in the nightly runs when azure-core@1.0.0 isn't published?

Reply:
I guess nightly run installs core using -e instead of from pypi?

Reply:
It depends how the nightly runs, it should build the wheels from the repo and use that as input. We discussed that with @scbedd already. :)